src/crypto/index.js

```diff
@@ -637,12 +637,13 @@ Crypto.prototype.bootstrapSecretStorage = async function({
             } else {
                 if (!newKeyId) {
                     logger.log("Secret storage default key not found, creating new key");
-                    const keyOptions = await createSecretStorageKey();
+                    const [keyOptions, , key] = await createSecretStorageKey();
```

Comment:
We should probably document the call signature wherever this gets defined.

Reply:
While working on this yesterday, I had wanted to change this to an object for easier use, but didn't want to make a breaking change... Today though, I noticed it's part of the unstable API surface for cross-signing.

I've changed this to return a object instead and updated docs to match.